modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/js/main.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+Liferay.on('portletReady', (event) => {
+	const {portletId} = event;
+
+	if (
+		!portletId.startsWith(
+			'com_liferay_account_admin_web_internal_portlet_AccountEntriesAdminPortlet'
+		)
+	) {
+		return;
+	}
+
+	const portletNamespace = '_' + portletId + '_';
+
+	window[`${portletNamespace}addDomains`] = (eventName) => {
+		const domainsInput = document.getElementById(
+			`${portletNamespace}domain`
+		);
+
+		const domains = domainsInput.value.split(',');
```

Comment:
We could, and probably should check if `domainsInput` exists here before using it